src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,76 @@
+
+Cockroach Labs recommends using the consistent cutover approach when using the live migration strategy. Your data remains more consistent and downtime is minimized. However, if zero downtime is truly required, the immediate cutover approach may be used at the risk of some data inconsistencies. Third-party migration tools like AWS DMS, Qlik, and Striim perform the initial migration into CockroachDB, and replicate ongoing changes. CockroachDB changefeeds allow for replication to the source database after cutover, simplifying the service architecture.
+
+## Advantages and disadvantages of the live migration strategy
+
+{{site.data.alerts.callout_info}}
+The information on this page assumes you have already reviewed the [migration overview]({% link {{ page.version.version }}/migration-overview.md %}).
+{{site.data.alerts.end}}
+
+When choosing a migration strategy, the fundamental question you should ask is: Do you want to fix problems related to the migration during the migration or afterward? The risks of migrating data are very similar, but the strategy you choose will determine when you will have to take the risk. Choosing the lift-and-shift strategy typically means fixing problems after the migration. Choosing live migration typically means fixing problems during the migration, before the cutover.
+
+On the spectrum of different data migration strategies, live migration has the following advantages and disadvantages. The terms "lower" and "higher" are not absolute, but relative to other approaches.
+
+Advantages:
+
+- There's greater service availability with live migrations. You can failback to the source database at any point in the migration process, or if there are significant errors after cutover.
+- The impact of a migration on users is minimized by having very little downtime.
+- There's less internal coordination cost. For example, if other teams use the database, they can update their systems to during the initial phase of the migration with no extended outage or maintenance window.
+- You can test the viability and scalability of your architecture on live production data. For example, you can tune the performance of CockroachDB before the cutover without affecting the performance of the production application.
+
+Disadvantages:
+
+- It is complex to implement. You need the right application design and replication tools to successfully perform the migration.
+- It requires abstracting the database operations in the application. You need complete control over all the applications reading and writing to the database to ensure the data is consistent across both the source and target database.
+- During migration there's less certainty about the integrity of the data due to multiple systems being updated.
```

Comment:
Basically, yes. E.g. interruptions to the replication service or transaction errors to one database during a dual-write scenario introduce inconsistency between the source and target databases. It doesn't necessarily have to be concurrently, though. If your replication service is run periodically instead of in real-time, for example.